website/content/docs/auth/azure.mdx

```diff
@@ -234,6 +234,63 @@ tool.
        https://127.0.0.1:8200/v1/auth/azure/role/dev-role
    ```
 
+## Root credential rotation
+
+If the mount is configured with credentials directly, the credential's key may be
+rotated to a Vault-generated value that is not accessible by the operator.
+This will ensure that only Vault is able to access the "root" user that Vault uses to
+manipulate dynamic & static credentials.
+
+```shell-session
+vault write -f auth/azure/rotate-root
+```
+
+### Schedule-based root credential rotation
+
+<EnterpriseAlert product="vault" />
```

Comment:
We have dedicated partials for enterprise alerts depending on whether the feature is available in HVD **and** enterprise or enterprise only: `@include 'alerts/enterprise-only.mdx'` or `@include 'alerts/enterprise-and-hcp.mdx`

Reply:
applied in 5 places across the 4 plugins (twice in auth/ldap)